### PR TITLE
player-cmus: show tags instead of filename, show position and duration

### PR DESCRIPTION
**Pull request by alexshoo**
Rewrote the script to show name and artist from tags instead of filename and show current position and track duration.
Also sending the '--toggle' option and handling it through case was unnecessary, as 'cmus-remote -u' already toggles the pause.

### CODE REVIEW

File: polybar-scripts/player-cmus/player-cmus.sh [hunk 1]
```diff
@@ -1,16 +1,26 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (x70b1) [inline polybar-scripts/player-cmus/player-cmus.sh]: Could you use `/bin/sh` and fix all `shellsheck` errors? I don't see why bash is explicitly needed here.

Comment (alexshoo) [reply polybar-scripts/player-cmus/player-cmus.sh]: Sure, will do!

File: polybar-scripts/player-cmus/player-cmus.sh [hunk 2]
```diff
@@ -1,16 +1,26 @@
 #!/bin/sh
+# shellcheck disable=SC2034
 
-case "$1" in
-    --toggle)
```
[the remark this answers is not in the document]
Comment (alexshoo) [reply polybar-scripts/player-cmus/player-cmus.sh]: As I mentioned, "cmus-remote -u" already toggles pause/play, so handling it through case isn't necessary. Am I missing something?

[the remark this answers is not in the document]
Comment (x70b1) [reply polybar-scripts/player-cmus/player-cmus.sh]: Oh, ok. I didn't know that exactly. I'm not a `cmus` user.
I guess that was all. :+1:

**Comment by x70b1** [comment]
Should we add different icons for playing and paused?

**Comment by x70b1** [comment]
@alexshoo I rewrote the script a bit.

**Comment by maxdevjs** [comment]
Some times 

```
cmus-remote -Q 2 "$info" | grep position | awk -F\  '{print $2}'
```
returns something similar to

```
23
comment
```
(23 being an example duration, followed by the word comment), which breaks the script outputting the name of the file. For what I understand, it happens when the archive has a comment beginning with `\n`, but without a `tag comment`.

When there is a `tag comment`, the script does not break and outputs a partial result (only artist, no title).

Some other times (with the `tag comment` or without it), the output is the expected.

**Pull request merged**